### PR TITLE
Added LinkedListCycle naive solution

### PR DESCRIPTION
**Pull request by PShchegolevatykh**
Finally "my level of difficulty" :D
leetcoderesult

### CODE REVIEW

File: pshchegolevatykh/LinkedListCycle.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            head = newHead;
+        }
+
+        public static bool HasCycle(ListNode head)
+        {
+            var visitedNodes = new List<ListNode>();
+
+            while(head != null)
+            {
+                if (visitedNodes.Contains(head))
```
Comment (rokovalenko) [inline pshchegolevatykh/LinkedListCycle.cs:66]: who uses list structure for search 
image

Comment (rokovalenko) [reply pshchegolevatykh/LinkedListCycle.cs:66]: although, memory usage is good somehow 🤔

Comment (vlemish) [reply pshchegolevatykh/LinkedListCycle.cs:66]: @quadreff typical hash set fan and list average enjoyer

image

Comment (rokovalenko) [reply pshchegolevatykh/LinkedListCycle.cs:66]: https://www.youtube.com/watch?v=Ux5cQbO_ybw

Comment (PShchegolevatykh) [reply pshchegolevatykh/LinkedListCycle.cs:66]: Yea somehow I managed to find data structure where insertion is the bottleneck. But leetcode said it's okay so I blame them for not checking this. So far it's the most NAIVE solution. Keep those memes coming!

Comment (makcgr) [reply pshchegolevatykh/LinkedListCycle.cs:66]: Actually Add() is O(1) for List<T> though Contains is definitely O(N) 🙂

**Pull request merged**